platform-tests/broker-acceptance/cdn_broker_test.go

```diff
@@ -40,17 +40,67 @@ var _ = Describe("CDN broker", func() {
 
 			serviceInstanceName = generator.PrefixedRandomName(testConfig.GetNamePrefix(), "test-cdn")
 
+			// best effort tidyup - we don't really care if these pass or fail
+			defer pollForServiceDeletionCompletion(serviceInstanceName)
+			defer cf.Cf("delete-domain", domainName, "-f")
+			defer cf.Cf("delete-service", serviceInstanceName, "-f")
+
 			By("creating a CDN instance: "+serviceInstanceName, func() {
 				Expect(cf.Cf("create-domain", orgName, domainName).Wait(testConfig.DefaultTimeoutDuration())).To(Exit(0))
 				Expect(cf.Cf("create-service", serviceName, serviceName, serviceInstanceName, "-c", domainNameList).Wait(testConfig.DefaultTimeoutDuration())).To(Exit(0))
 				pollForCdnServiceCreationCompletion(serviceInstanceName)
 			})
 
-			defer By("deleting a standard CDN service", func() {
+			By("deleting a standard CDN service", func() {
 				Expect(cf.Cf("delete-service", serviceInstanceName, "-f").Wait(testConfig.DefaultTimeoutDuration())).To(Exit(0))
 				Expect(cf.Cf("delete-domain", domainName, "-f").Wait(testConfig.DefaultTimeoutDuration())).To(Exit(0))
 				pollForServiceDeletionCompletion(serviceInstanceName)
 			})
 		})
+
+		It("refuses to create a CDN for a domain without a cf domain", func() {
```

Comment:
Without a **cf create-domain** no?

Reply:
I think I was trying to noun-ify it a bit but yeah it makes more sense to just say `cf create-domain`